pages/index.tsx

```diff
@@ -1,54 +1,55 @@
-import { ICustomer } from '@cpg/Interfaces/Customer.interface';
-import { useSession } from 'next-auth/react';
-import {Component, useEffect, useState} from 'react';
+import {ICustomer} from '@cpg/Interfaces/Customer.interface';
+import type {NextPage} from 'next'
+import {useSession} from 'next-auth/react';
+import {useEffect, useState} from 'react';
 import Loading from "../components/Loading";
 
-class Home extends Component
+const Home: NextPage = () =>
```

Comment:
This change caused the webapp to just **NOT** work.
After this PR it works fine.